src/EditorFeatures/VisualBasicTest/Recommendations/RecommendationTestHelpers.vb

```diff
@@ -50,16 +51,25 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Recommendations
             Assert.Equal(expectedText, recommendedKeyword.DescriptionFactory(CancellationToken.None).GetFullText())
         End Function
 
+        Friend Async Function VerifyRecommendationsWithPriority(testSource As XElement, priority As Integer, ParamArray recommendations As String()) As Task
+            Assert.NotEmpty(recommendations)
+
+            Dim source = ConvertTestSourceTag(testSource)
+
+            Await VerifyRecommendationsContainNothingTypedAsync(source, priority, recommendations).ConfigureAwait(False)
+            Await VerifyRecommendationsContainPartiallyTypedAsync(source, priority, recommendations).ConfigureAwait(False)
+        End Function
+
         Friend Async Function VerifyRecommendationsContainAsync(testSource As XElement, ParamArray recommendations As String()) As Task
             Assert.NotEmpty(recommendations)
 
             Dim source = ConvertTestSourceTag(testSource)
 
-            Await VerifyRecommendationsContainNothingTypedAsync(source, recommendations)
-            Await VerifyRecommendationsContainPartiallyTypedAsync(source, recommendations)
+            Await VerifyRecommendationsContainNothingTypedAsync(source, Nothing, recommendations)
```

Comment:
This is awkward: I can't add named args because the `recommendations` param is a params array and it's an error to name a params array argument :(
![image](https://cloud.githubusercontent.com/assets/3751401/15909414/2b893850-2d7b-11e6-95ea-074062049b68.png)